windows下运行haloServer和watch出错

容器内报错，详细日志见附件
[haloServer.log](https://github.com/halo-sigs/halo-gradle-plugin/files/11654922/haloServer.log)
```
2023-06-06T00:54:28.592+08:00  INFO 7 --- [         task-1] run.halo.app.search.IndicesInitializer   : Initialize post indices...
2023-06-06T00:54:28.635+08:00 ERROR 7 --- [           main] o.s.boot.SpringApplication               : Application run failed

java.lang.IllegalArgumentException: Path component should be '/'
	at java.base/sun.nio.fs.UnixFileSystemProvider.checkUri(Unknown Source) ~[na:na]
	at java.base/sun.nio.fs.UnixFileSystemProvider.getFileSystem(Unknown Source) ~[na:na]
	at java.base/java.nio.file.FileSystems.getFileSystem(Unknown Source) ~[na:na]
	at org.pf4j.util.FileUtils.getFileSystem(FileUtils.java:280) ~[pf4j-3.9.0.jar:3.9.0]
	at org.pf4j.util.FileUtils.getPath(FileUtils.java:245) ~[pf4j-3.9.0.jar:3.9.0]
	at org.pf4j.util.FileUtils.getPath(FileUtils.java:241) ~[pf4j-3.9.0.jar:3.9.0]
	at run.halo.app.plugin.YamlPluginFinder.getManifestPath(YamlPluginFinder.java:122) ~[classes/:2.5.2]
	at run.halo.app.plugin.YamlPluginFinder.readPluginDescriptor(YamlPluginFinder.java:78) ~[classes/:2.5.2]
	at run.halo.app.plugin.YamlPluginFinder.find(YamlPluginFinder.java:65) ~[classes/:2.5.2]
	at run.halo.app.plugin.PluginDevelopmen
tInitializer.createFixedPluginIfNecessary(PluginDevelopmentInitializer.java:47) ~[classes/:2.5.2]
	at run.halo.app.plugin.PluginDevelopmentInitializer.onApplicationEvent(PluginDevelopmentInitializer.java:42) ~[classes/:2.5.2]
	at run.halo.app.plugin.PluginDevelopmentInitializer.onApplicationEvent(PluginDevelopmentInitializer.java:20) ~[classes/:2.5.2]
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:176) ~[spring-context-6.0.8.jar:6.0.8]
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:169) ~[spring-context-6.0.8.jar:6.0.8]
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:143) ~[spring-context-6.0.8.jar:6.0.8]
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:413) ~[spring-context-6.0.8.jar:6.0.8]
	at org.springframewor
k.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:370) ~[spring-context-6.0.8.jar:6.0.8]
	at org.springframework.boot.context.event.EventPublishingRunListener.ready(EventPublishingRunListener.java:109) ~[spring-boot-3.0.6.jar:3.0.6]
	at org.springframework.boot.SpringApplicationRunListeners.lambda$ready$6(SpringApplicationRunListeners.java:80) ~[spring-boot-3.0.6.jar:3.0.6]
	at java.base/java.lang.Iterable.forEach(Unknown Source) ~[na:na]
	at org.springframework.boot.SpringApplicationRunListeners.doWithListeners(SpringApplicationRunListeners.java:118) ~[spring-boot-3.0.6.jar:3.0.6]
	at org.springframework.boot.SpringApplicationRunListeners.doWithListeners(SpringApplicationRunListeners.java:112) ~[spring-boot-3.0.6.jar:3.0.6]
	at org.springframework.boot.SpringApplicationRunListeners.ready(SpringApplicationRunListeners.java:80) ~[spring-boot-3.0.6.jar:3.0.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:329) ~[spring-boot-3.0.6.jar:3.0.6]
	at 
org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:150) ~[spring-boot-3.0.6.jar:3.0.6]
	at run.halo.app.Application.main(Application.java:28) ~[classes/:2.5.2]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[na:na]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[na:na]
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49) ~[application/:na]
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:95) ~[application/:na]
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58) ~[application/:na]
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:65) ~[application/:na]
2023-06-06T00:54:28.646+08:00  INFO 7 --- [           main] r.h.a.e.controller.DefaultController     : Disposing controller run.halo.app.metrics.VisitedEventReconciler
2023-06-06T00:54:28.647+08:00  INFO 7 --- [tReconciler-t-1] r.h.a.e.controller.DefaultController     : Controller worker run.halo.app.metrics.VisitedEventReconciler-worker-1 interrupted
```
```
2023-06-06T00:54:28.659+08:00  INFO 7 --- [           main] r.h.a.e.controller.DefaultController     : Controller run.halo.app.metrics.VotedEventReconciler is disposed
2023-06-06T00:54:28.665+08:00 ERROR 7 --- [         task-1] .a.i.SimpleAsyncUncaughtExceptionHandler : Unexpected exception occurred invoking async method: public void run.halo.app.search.IndicesInitializer.whenSchemeInitialized(run.halo.app.infra.SchemeInitializedEvent) throws java.lang.InterruptedException

java.lang.InterruptedException: null
	at java.base/java.util.concurrent.locks.AbstractQueuedSynchronizer.acquireSharedInterruptibly(Unknown Source) ~[na:na]
	at java.base/java.util.concurrent.CountDownLatch.await(Unknown Source) ~[na:na]
	at run.halo.app.search.IndicesInitializer.whenSchemeInitialized(IndicesInitializer.java:31) ~[classes/:2.5.2]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[na:na]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[na:na]
	at org.sprin
gframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:343) ~[spring-aop-6.0.8.jar:6.0.8]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.0.8.jar:6.0.8]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.0.8.jar:6.0.8]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:750) ~[spring-aop-6.0.8.jar:6.0.8]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-6.0.8.jar:6.0.8]
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[na:na]
	at java.base/java.lang.Thread.run(Unknown Source) ~[na:na]
2023-06-06T00:54:28.667+08:00  INFO 7 --- [           main] r.h.a.e.c.DefaultControllerManager       : Shutting down 19 controllers...
2023-06-06T00:54:28.669+08:00  INFO 7 --- [           main] r.h.a.e.c.DefaultControllerManager       : Shutting down controller run.halo.app.infra.SetupStateCache...
```
```
2023-06-06T00:54:28.674+08:00  INFO 7 --- [           main] r.h.a.e.controller.DefaultController     : Disposing controller run.halo.app.core.extension.reconciler.SystemSettingReconciler
2023-06-06T00:54:28.677+08:00 ERROR 7 --- [gReconciler-t-1] r.h.a.e.controller.DefaultController     : Reconciler in run.halo.app.core.extension.reconciler.SystemSettingReconciler-worker-1 aborted with an error, re-enqueuing...

reactor.core.Exceptions$ReactiveException: java.lang.InterruptedException
	at reactor.core.Exceptions.propagate(Exceptions.java:408) ~[reactor-core-3.5.5.jar:3.5.5]
	at reactor.core.publisher.BlockingOptionalMonoSubscriber.blockingGet(BlockingOptionalMonoSubscriber.java:112) ~[reactor-core-3.5.5.jar:3.5.5]
	at reactor.core.publisher.Mono.blockOptional(Mono.java:1755) ~[reactor-core-3.5.5.jar:3.5.5]
	at run.halo.app.infra.SystemConfigurableEnvironmentFetcher.getConfigMapBlocking(SystemConfigurableEnvironmentFetcher.java:98) ~[classes/:2.5.2]
	at run.halo.app.core.extension.reconciler.SystemSettingReconciler.getConfigMap(SystemSettingReconciler.java:298) ~[classes/:2.5.2]
	at run.halo.app.core.extension.reconciler.SystemSettingReconciler$RouteRuleReconciler.reconcileTagsRule(SystemSetti
ngReconciler.java:182) ~[classes/:2.5.2]
	at run.halo.app.core.extension.reconciler.SystemSettingReconciler$RouteRuleReconciler.reconcile(SystemSettingReconciler.java:126) ~[classes/:2.5.2]
	at run.halo.app.core.extension.reconciler.SystemSettingReconciler.lambda$reconcile$0(SystemSettingReconciler.java:60) ~[classes/:2.5.2]
	at java.base/java.util.Optional.ifPresent(Unknown Source) ~[na:na]
	at run.halo.app.core.extension.reconciler.SystemSettingReconciler.reconcile(SystemSettingReconciler.java:58) ~[classes/:2.5.2]
	at run.halo.app.core.extension.reconciler.SystemSettingReconciler.reconcile(SystemSettingReconciler.java:31) ~[classes/:2.5.2]
	at run.halo.app.extension.controller.DefaultController$Worker.run(DefaultController.java:163) ~[api-2.5.2.jar:na]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[na:na]
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[na:na]
	at
 java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[na:na]
	at java.base/java.lang.Thread.run(Unknown Source) ~[na:na]
	Suppressed: java.lang.Exception: #blockOptional() has been interrupted
		at reactor.core.publisher.BlockingOptionalMonoSubscriber.blockingGet(BlockingOptionalMonoSubscriber.java:114) ~[reactor-core-3.5.5.jar:3.5.5]
		... 15 common frames omitted
Caused by: java.lang.InterruptedException: null
	at java.base/java.util.concurrent.locks.AbstractQueuedSynchronizer.acquireSharedInterruptibly(Unknown Source) ~[na:na]
	at java.base/java.util.concurrent.CountDownLatch.await(Unknown Source) ~[na:na]
	at reactor.core.publisher.BlockingOptionalMonoSubscriber.blockingGet(BlockingOptionalMonoSubscriber.java:108) ~[reactor-core-3.5.5.jar:3.5.5]
	... 15 common frames omitted
2023-06-06T00:54:28.679+08:00  INFO 7 --- [gReconciler-t-1] r.h.a.e.controller.DefaultController     : Controller worker run.halo.app.core.extension.reconciler.SystemSettingReconciler-worker-1 is stopped
2023-06-06T00:54:28.679+08:00  INFO 7 --- [           main] r.h.a.e.controller.DefaultController     : Controller run.halo.app.core.extension.reconciler.SystemSettingReconciler is disposed
2023-06-06T00:54:28.680+08:00  INFO 7 --- [           main] r.h.a.e.c.DefaultControllerManager       : Shutdown controller run.halo.app.core.extension.reconciler.SystemSettingReconciler successfully
2023-06-06T00:54:28.681+08:00  INFO 7 --- [           main] r.h.a.e.c.DefaultControllerManager       : Shutting down controller run.halo.app.core.extension.reconciler.RoleReconciler...
2023-06-06T00:54:28.681+08:00  INFO 7 --- [           main] r.h.a.e.controller.DefaultController     : Disposing controller run.halo.app.core.extension.reconciler.RoleReconciler
2023-06-06T00:54:28.683+08:00 ERROR 7 --- [eReconciler-t-1] r.h.a.e.controller.DefaultController     : Reconciler in run.halo.app.core.extension.reconciler.RoleReconciler-worker-1 aborted with an error, re-enqueuing...

reactor.core.Exceptions$ReactiveException: java.lang.InterruptedException
	at reactor.core.Exceptions.propagate(Exceptions.java:408) ~[reactor-core-3.5.5.jar:3.5.5]
	at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:91) ~[reactor-core-3.5.5.jar:3.5.5]
	at reactor.core.publisher.Mono.block(Mono.java:1710) ~[reactor-core-3.5.5.jar:3.5.5]
	at run.halo.app.extension.DelegateExtensionClient.update(DelegateExtensionClient.java:52) ~[classes/:2.5.2]
	at run.halo.app.core.extension.reconciler.RoleReconciler.lambda$updateLabelsAndAnnotations$2(RoleReconciler.java:62) ~[classes/:2.5.2]
	at java.base/java.util.Optional.ifPresent(Unknown Source) ~[na:na]
	at run.halo.app.core.extension.reconciler.RoleReconciler.updateLabelsAndAnnotations(RoleReconciler.java:59) ~
[classes/:2.5.2]
	at run.halo.app.core.extension.reconciler.RoleReconciler.lambda$reconcile$0(RoleReconciler.java:41) ~[classes/:2.5.2]
	at java.base/java.util.Optional.ifPresent(Unknown Source) ~[na:na]
	at run.halo.app.core.extension.reconciler.RoleReconciler.reconcile(RoleReconciler.java:35) ~[classes/:2.5.2]
	at run.halo.app.core.extension.reconciler.RoleReconciler.reconcile(RoleReconciler.java:22) ~[classes/:2.5.2]
	at run.halo.app.extension.controller.DefaultController$Worker.run(DefaultController.java:163) ~[api-2.5.2.jar:na]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[na:na]
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[na:na]
	at java.base/java.lang.Thread.run(Unknown Source) ~[na:na]
Caused by: java.lang.InterruptedException: null
	at java.base/java.util.concurre
nt.locks.AbstractQueuedSynchronizer.acquireSharedInterruptibly(Unknown Source) ~[na:na]
	at java.base/java.util.concurrent.CountDownLatch.await(Unknown Source) ~[na:na]
	at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:87) ~[reactor-core-3.5.5.jar:3.5.5]
	... 15 common frames omitted
2023-06-06T00:54:28.684+08:00  INFO 7 --- [eReconciler-t-1] r.h.a.e.controller.DefaultController     : Controller worker run.halo.app.core.extension.reconciler.RoleReconciler-worker-1 is stopped
2023-06-06T00:54:28.685+08:00  INFO 7 --- [           main] r.h.a.e.controller.DefaultController     : Controller run.halo.app.core.extension.reconciler.RoleReconciler is disposed
```

--- a/src/main/java/run/halo/gradle/docker/DockerCreateContainer.java
+++ b/src/main/java/run/halo/gradle/docker/DockerCreateContainer.java
@@ -183,7 +183,7 @@ public class DockerCreateContainer extends DockerExistingImage {
                 + "address=*:" + debugPort);
         }
         envs.add("HALO_PLUGIN_RUNTIMEMODE=development");
-        envs.add("HALO_PLUGIN_FIXEDPLUGINPATH=" + Paths.get(buildPluginDestPath(pluginName)));
+        envs.add("HALO_PLUGIN_FIXEDPLUGINPATH=" + buildPluginDestPath(pluginName));
         envs.add("HALO_WORKDIR=" + haloWorkDir());
         String configLocation = PathUtils.combinePath(haloWorkDir(), "/config") + "/";
         envs.add(

